.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules/
 config/development.js
+taskcluster-secrets.conf.json
```

Comment:
usually I store local test config in a file like this...

But we'll have the nice JSON config soon :) I really hope...